services/external_logging/supervisor.conf

```diff
@@ -0,0 +1,7 @@
+[program:cron]
+command=/usr/bin/env python baselayer/services/cron/cron.py %(ENV_FLAGS)s
+environment=PYTHONPATH=".",PYTHONUNBUFFERED="1"
+startretries=0
+startsecs=1
+stdout_logfile=log/cron.log
```

Comment:
These need to be updated as they appear to be copied & unchanged from the cron service (program name, command, stdout_logfile)

Reply:
Good catch. Strange -- I had the correct file in there in my repo. Must have checked in the wrong version. Fixed now.